content/en/about-filecoin/managing-assets/index.md

```diff
@@ -9,13 +9,31 @@ weight: 60
 
 ## Wallets
 
+<!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->
+Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```

Comment:
```suggestion
`FIL` is the native token of the Filecoin network. FIL tokens are used as an economic incentive to ensure data is stored reliably over time.

```
```suggestion
Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```
Let's leave the wrapped FIL reference out, people can find it in the sidebar if desired.